README.md

```diff
@@ -1 +1,46 @@
-# chpr-newrelic
\ No newline at end of file
+# chpr-newrelic
+
+## Purpose of this library
+
+The purpose of this library is to make a consistent integration of newrelic
+in all node apps as simple as possible, and to have to modify all the projects
+as rarely as possible.
+
+This library is here to centralize the configuration of newrelic instead of
+duplicating it across all node applications
+
+## Configuration
+
+Your app must have these environment variables defined to use this library
+
+**NEW_RELIC_ENABLED**: `"true"` or `"false"` (default `"true"`). Defines whether
+newrelic should be used or not. If set to `"false"`, newrelic won't be
+loaded at all.
+
+**NEW_RELIC_LICENSE_KEY**: The licence key to use with newrelic.
+
+**NEW_RELIC_APP_NAME**: The application name in newrelic
+
+Don't use any other newrelic configuration variables
+
+## Use
+
+Insert this line at the very top of the entry point of the process.
+
+    require('chpr-newrelic');
+
+Don't put a newrelic.js file in your project
```

Comment:
It won't break anything it the file is here. It will just be ignored. So it's not bad enough to put it in uppercase according to my standards.